arch/arm/src/sama5/sam_tsd.c

```diff
@@ -249,8 +249,8 @@ static const struct file_operations g_tsdops =
   NULL,            /* write */
   NULL,            /* seek */
   sam_tsd_ioctl,   /* ioctl */
-  NULL,            /* truncate */
   NULL,            /* mmap */
+  NULL,            /* truncate */
   sam_tsd_poll     /* poll */
 #ifndef CONFIG_DISABLE_PSEUDOFS_OPERATIONS
   , NULL           /* unlink */
```

Comment:
@xiaoxiang781216 could you please rebase the PR? I think all those ending NULLs should be removed now.

Reply:
Done.